kratos/python_scripts/vtk_output_process.py

```diff
@@ -9,6 +9,7 @@ def Factory(settings, Model):
 
 ## All the processes python should be derived from "Process"
 class VtkOutputProcess(KratosMultiphysics.Process):
+    "IMPORTANT : when output_control_type is time, then paraview will not be able to group them "
```

Comment:
I didn't know that
Do you still want to support it then?

Reply:
It just that Paraview cannot animate them in series. But it can surely display the result which is output.